### DISCUSSION TITLE
Error when run app.py

### DISCUSSION THREAD

**Discussion by yonvisatria20** [teamalgoritma/community#191] [Q&A] [answered]
![Screenshot (368)](https://user-images.githubusercontent.com/99859250/178138834-489595dc-5d63-4545-90d0-c97a48fd456e.png)

OverflowError : int too big to convert
How to fix this error? In my data I have millions number and int64 can't to convert.

My code :

```
df = pd.DataFrame(temp, columns =('Date', 'Volume'))

#insert data wrangling here
df['Volume'] = df['Volume'].str.replace(',' , '')
df['Volume'] = df['Volume'].str.replace('$', '')
df['Volume'] = df['Volume'].astype('int64')

```

**Comment by triarm** [comment, marked as answer]
Hi @yonvisatria20 ,
Sebelumnya apakah Anda dapat melampirkan semua error dari hasil deploy tersebut agar kami dapat lebih memahami error yang terjadi? Selain itu apakah Anda dapat mencari nilai maksimum pada data Anda untuk melihat apakah nilainya terlalu besar/tidak?

Demikian yang dapat kami sampaikan
